packages/components/src/components/primitives/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+import {
+  Avatar as ChakraAvatar,
+  AvatarBadge as ChakraAvatarBadge,
+  useStyleConfig,
+} from '@chakra-ui/react';
+import { AvatarProps, AvatarBadgeProps, AvatarBadgeStyle } from './types';
+
+export const Avatar: FC<AvatarProps> = ({ showBadge, size, ...rest }) => (
+  <ChakraAvatar size={size} {...rest}>
+    {showBadge && <AvatarBadge size={size} />}
+  </ChakraAvatar>
+);
+
+const AvatarBadge: FC<AvatarBadgeProps> = ({ size, ...rest }) => {
+  const styles = useStyleConfig('AvatarBadge', { size }) as AvatarBadgeStyle;
+  return (
+    <ChakraAvatarBadge data-testid="cmpsr.avatar.badge" {...styles} {...rest} />
+  );
+};
```

Comment:
Feel free to ignore this suggestion...

What do you think of defining the default values at the parameters level?

```suggestion
const AvatarBadge: FC<AvatarBadgeProps> = ({ size, testId="cmpsr.avatar.badge", ...rest }) => {
  const styles = useStyleConfig('AvatarBadge', { size }) as AvatarBadgeStyle;
  return (
    <ChakraAvatarBadge data-testid={testId} {...styles} {...rest} />
  );
};
```